src/main/java/com/vlf/intentsender/Models/Command.java

```diff
@@ -1,24 +1,23 @@
-package intentsender.Models;
+package com.vlf.intentsender.Models;
```

Comment:
Just wonder if the package name should use Distillery's website address instead:
```suggestion
package com.distillery.intentsender.Models;
```

Reply:
Wanted something short. But does not really matter for me. Will probably update